deploy/rbac-clowdapp.yml

```diff
@@ -631,6 +631,19 @@ objects:
           volumeMounts:
           - mountPath: /tmp
             name: users-data
+      - name: seed-roles-perms-groups
+        podSpec:
+          image: quay.io/cloudservices/rbac:latest
+          command:
+            - ./rbac/manage.py seeds
+            - --force-create
```

Comment:
You probably need to separate `seeds` part in its own element?

And does this need `--roles` ? I guess not, but maybe helps clarify our intent.

Reply:
I was taking reference from the job above `- ./rbac/manage.py import_tenant_user_data` which seems to be just the filename, couldn't find other references for it. It doesn't require `--roles`, but added to clarify!